open-subshare/open-subshare-json-contract.md

```diff
@@ -30,14 +30,32 @@ Poweron response:
     "categories": [
       {
         "name": "Certificate of deposit",
-        "groupNumber": "??",
+        "groupNumber": "0",
         "accountTypes": [
           {
-            "shareType": "??",
+            "shareType": "0",
             "name": "7 Month certificate",
             "term": "7 months",
             "minBalance": "1000.00",
-            "interestRate": "2.450%" // optional
+            "interestRates": [
+              {
+                "rate": "0.000",
+                "minBalance": "0.00",
+                "maxBalance": "4.99",
```

Comment:
I think the comma after the "maxBalance" values isn't needed because that's the last element in the group